client/src/frontend/components/journal/index.js

```diff
@@ -0,0 +1,9 @@
+import IssueList from "./IssueList";
```

Comment:
Just a question about how we're organizing components: do these not live under `composed` because they're at the level of a page layout?

Reply:
Since these are specific to journals, I think it makes sense to keep them in a `/journals` dir. I think `/composed` should be limited to model-agnostic "Entity" components so we don't end up just putting every component we build there. How's that sound?